ui/app/services/permissions.js

```diff
@@ -108,33 +108,53 @@ export default Service.extend({
     }
   },
 
-  hasPermission(pathName) {
+  hasPermission(pathName, capabilities = [null]) {
```

Comment:
Why `[null]` instead of `[]`?

Reply:
setting capabilities as `[]` caused the tests to fail & incorrectly show items because calling `.every` on an empty array returned `true`. javascript! 🤷‍♀️

Reply:
huh guess I haven't used `every`/`some` enough to realize that - thanks!